kin-sdk/src/main/java/kin/sdk/Environment.java

```diff
@@ -14,7 +14,7 @@
 			"Kin Mainnet ; December 2018");
 
     public static final Environment TEST =
-		new Environment("https://horizon-testnet.kininfrastructure.com/",
+        new Environment("http://haproxy.kininfrastructure.com/",
```

Comment:
why it's not https? and why it's called like that....
does blockchain wait for something before adding ssl?